pkg/controller/egress/controller.go

```diff
@@ -30,17 +30,19 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/retry"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/klog/v2"
 
 	"antrea.io/antrea/pkg/apis/controlplane"
-	egressv1alpha2 "antrea.io/antrea/pkg/apis/crd/v1alpha2"
+	. "antrea.io/antrea/pkg/apis/crd/v1alpha2"
```

Comment:
This is not an encouraged usage, which misleads people that the types are in same package. Package qualifier is very useful to indicate where the dependency is. The code shouldn't pretend they are in same package which makes dividing them in different packages pointless.

Reply:
Done